.gitignore

```diff
@@ -0,0 +1,120 @@
+#Liscense
+LISCENSE
+Liscense
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```

Comment:
This app doesn’t contain these all files. Please make a commit with gitignore file removed.

Reply:
ok